e2e/README.md

```diff
@@ -0,0 +1,21 @@
+# end-to-end tests
+
+## Install cypress
+
+```
+npm install
```

Comment:
I'd prefer if developers don't have to install npm on their machine. The idea behind the docker setup is that only docker needs to be installed on a dev computer. This way we don't have to worry about different npm versions (like v6 and v7 wrote different lockfile formats), and we could even easily switch to yarn at any time. Could we provide a docker container which is officially used to run the e2e tests locally?
In qualix, I included a [service](https://github.com/gloggi/qualix/blob/master/docker-compose.yml#L50) in docker-compose.yml which exits immediately when running `docker-compose up` normally, but can be used to run the e2e tests or open the cypress UI with a `docker-compose run` [command](https://github.com/gloggi/qualix#end-to-end-tests). If the extra service in docker-compose.yml and in docker-compose ps bothers you, we could instead add a separate e2e-tests.yml which could be invoked like `docker-compose -f e2e-tests.yml up`, or at least include a `docker run` command as the default recommended option for new hobbyist devs in this readme.

Reply:
+1 for this

Reply:
One could run the tests via docker with the same command that was previously in the github workflow:
```
docker run -v $PWD:/e2e -w /e2e --network host cypress/included:12.7.0
```

I can add this to the README. Or would you prefer a docker compose service?

Reply:
Okay for me :+1:

Reply:
https://github.com/ecamp/ecamp3/pull/3309/commits/f36b0be8ba77c866d56d3488bd877e940a514ab8

Reply:
Actually, since #3305, we have a docker compose profile in our docker-compose.yml. So we already have the translation service in docker-compose.yml for only running the translation script sometimes. I would propose we use the same mechanism for the e2e tests.
(Actually, right now our docker-compose.yml is invalid, because profiles are only supported starting with docker-compose.yml version 3.9. But I'm fixing this right now and updating the linux setup instructions, since the recommended way to install Docker on Linux has changed)